cos-fleet-manager-api/src/openapi/specs/connector_mgmt.yaml

```diff
@@ -1286,12 +1286,12 @@ components:
           id: "9bsv0s6brfr002pfnkh0"
           client_id: "srvc-acct-162ef2d8-0209-4117-8462-df63c2025c26"
           client_secret: "b144d991-08eb-4472-8ec1-eb034c69231b"
-          url: "foo-9bsv0s6brfr002pfnkh0.kas.acme.com:443"
+          url: "https://bu98.serviceregistry.rhcloud.com/t/51eba005-daft-punk-afe1-b2178bcb523d/apis/registry/v2"
```

Comment:
this needs to be updated in the fleet manager side, here the openapi definition is only used for generating the code.

Reply:
I know. But I need to do this with a separate PR. Some more things might diverge and I will have a look next week.

Reply:
Do we have an idea how we will receive SR URLs? Will it be a full URL like this?

Reply:
I see. Okay, this is a customized URL from the RHOSR UI. Then this example should be good!